src/content-scripts/components/AICodeRefactor/ChangeSuggestorButton.ts

```diff
@@ -36,11 +38,9 @@ const handleSubmit = async (commentNode: HTMLElement) => {
         if (!descriptionConfig) {
             return;
         }
-        if (!descriptionConfig.enabled) {
-            return alert("AI PR description is disabled!");
-        }
```

Comment:
Do we assume that the extension AI is always enabled?

Reply:
We need to remove these lines.

Reply:
This was removed but you missed some other lines. 
I will check it (found the missing lines). 
I was wondering, why we assume that the AI is disabled always?

Reply:
> This was removed but you missed some other lines.

What do you mean? It works after simply removing these lines.

> I was wondering, why we assume that the AI is disabled always?

We don't do enable/disable handling through AI configuration anymore, we simply don't inject the button in the DOM here - https://github.com/open-sauced/ai/blob/beta/src/utils/dom-utils/addDescriptionGenerator.ts#L18

Reply:
I meant the links I mentioned in this comment were removed, it was in the ChangeSuggestorButton :) 

I thought it would be needed to have the check for paid users, that's what made me feel confused about it.